.stylelintrc

```diff
@@ -4,7 +4,8 @@
     "./export/css/sass/**/*",
     "./source/sass/fonts/**/*",
     "./source/sass/variables/**/*",
-    "./source/sass/vendor/**/*"
+    "./source/sass/vendor/**/*",
+    "./test/sass/**/*"
```

Comment:
Ideally we want to skip certain rules inside `test`, rather than disable it completely (most of the standards still apply). I've tried in the past without success. But, the Gulpfile doesn't run on `test` anyway, so it's only IDEs that pick it up.